new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainOnPV.java

```diff
@@ -0,0 +1,700 @@
+            domainUid, domainNamespace));
+
+    // wait for the domain object to get created
+    logger.info("Checking for domain custom resouce object in namespace {0}", domainNamespace);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
+                + "(elapsed time {2}ms, remaining time {3}ms)",
+                domainUid,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(domainExists(domainUid, DOMAIN_VERSION, domainNamespace));
+
+    // check admin server pod is ready
+    logger.info("Waiting for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReady(adminServerPodName);
+
+    // check managed server pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
+          managedServerPodNamePrefix + i, domainNamespace);
+      checkPodReady(managedServerPodNamePrefix + i);
+    }
+
+    logger.info("Check admin service {0} is created in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkServiceCreated(adminServerPodName);
+
+    // check managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check managed server service {0} is created in namespace {1}",
+          managedServerPodNamePrefix + i, domainNamespace);
+      checkServiceCreated(managedServerPodNamePrefix + i);
+    }
+
+    logger.info("Validating WebLogic admin server access by login to console");
+    logger.info("Getting node port");
+    int serviceNodePort = assertDoesNotThrow(()
+        -> TestActions.getServiceNodePort(domainNamespace, adminServerPodName + "-external", "default"),
+        "Accessing admin server node port failed");
+    String consoleUrl = new StringBuffer()
+        .append("http://")
+        .append(K8S_NODEPORT_HOST)
+        .append(":")
+        .append(serviceNodePort)
+        .append("/console/login/LoginForm.jsp").toString();
+
+    assertDoesNotThrow(() -> loginTest(consoleUrl, adminUser, adminPassword), "Console login failed");
+  }
+
+  /**
+   * Creates a WebLogic domain on a persistent volume by doing the following.
+   * Copies the WLST domain script to a temp location.
+   * Creates a domain properties in the temp location.
+   * Creates a configmap containing domain scripts and property files.
+   * Runs a job to create domain on persistent volume.
+   * @throws IOException when reading/writing domain scripts fails
+   */
+  private void createDomainOnPV() throws IOException {
+
+    logger.info("create a staging location for domain creation scripts");
+    Path pvTemp = Paths.get(WORK_DIR, "ItDomainOnPV", "domainCreateTempPV");
```

Comment:
Can you use TestConstants.PV_ROOT? I know already know this location will work in our various clusters.

Reply:
Oh, nevermind... I've misread what this directory is for...

Reply:
I still might like this then in a directory under RESULT_ROOT so the script you generate is saved to look at after the test.

Reply:
Changed to use RESULTS_ROOT